python/setup.cfg

```diff
@@ -42,4 +42,7 @@ numpy =
 scikit-learn = 
 	scikit-learn
 	%(numpy)s
+polars =
+	polars==0.19.3 
```

Comment:
This is an unfortunate requirement. The serialization format for lazyframes needs to be equivalent between the polars and OpenDP libraries, so that lazyframes serialized from one library can be deserialized in the other.